src/commands/settings/AboutCommand.ts

```diff
@@ -38,16 +38,14 @@ export default class AboutCommand extends BaseCommand {
                 new MessageEmbed()
                 .setAuthor({ iconURL: client.user?.displayAvatarURL(), name: "SudoBot" })
                 .setDescription(`
-                    A free and open source Discord moderation bot, specially created for **The Everything Server**.
-
-                    Copyright (C) ${new Date().getFullYear()} OSN Inc.
```

Comment:
Oh wait, you removed the `Copyright (C) ${new Date().getFullYear()} OSN Inc.` part. You should keep that. I know the footer already has it but still. You can add it after the newline you've added. After this:

```diff
+  __**A free and open source Discord moderation bot, specially created for The Everything Server**__.\n
```